test/spec_helper.rb

```diff
@@ -38,3 +38,15 @@
     end
   end
 end
+
+RSpec.shared_examples "test_scripts" do |name|
+  list_path =  File.join(File.dirname(__FILE__),"../test", "#{name}.list")
+
+  File.readlines(list_path).map(&:chomp).each do |line|
+    test, description = line.split("#")
```

Comment:
This split does not work correctly for descriptions containing `#`,  e.g. with `bnc#xxxxxx`, use `split("#", 2) to limit it to max. 2 parts.